src/components/KymaModules/support.ts

```diff
@@ -61,6 +66,58 @@ export function useModuleStatus(resource: KymaResourceType) {
   return { data, loading, error };
 }
 
+export function useModulesStatuses(modules: any[]) {
```

Comment:
we have 2 functions, one to get status for one module, and this to get all of the statuses, please rename this function to be more different in naming than the 1 one. Maybe `useGetAllModulesStatuses` or `useAllModulesStatuses`